orchestrator/client/src/main.rs

```diff
@@ -103,10 +103,14 @@ async fn main() {
             amount,
             denom: peggy_denom.clone(),
         };
+        let bridge_fee = Coin {
```

Comment:
Good catch here, could you remove the user fee argument from the cosmos-to-eth string above?

Reply:
You mean eth-to-cosmos? Code uses fee argument for cosmos-to-eth.

Reply:
yeha I see in the batch request. I wonder if it's worth keeping put the pull request at least makes sense. I'm going to merge these. That's reasonable to clean up later. Thanks for your patience and contribution!